public/index.html

```diff
@@ -4,10 +4,22 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-
-  <title>Project Title</title>
+  <title>Clock</title>
+  <link href='https://fonts.googleapis.com/css?family=Averia+Sans+Libre:700' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" href="/normalize.css">
   <link rel="stylesheet" href="app.css">
 </head>
 <body>
+  <div class="clock">
+    <div class="background__color">
+      <div class="time">
```

Comment:
I think you meant for the `time` class to be on the `span` inside of this. Right now you are wiping the `.color` element out of the active DOM when you set the `innerHTML` of `.time`. This is what's leading to some issues.

You may want to add some extra selectors or move some class names around to get the end result.